tests/helpers/test_plates_helpers.py

```diff
@@ -1155,140 +1071,137 @@ def test_construct_cherrypicking_plate_failed_message_samples_not_in_mongo(
+#                 test_robot_serial_number = "12345"
+#                 test_failure_type = any_failure_type(app)
+#                 errors, _ = construct_cherrypicking_plate_failed_message(
+#                     test_barcode, test_user, test_robot_serial_number, test_failure_type
+#                 )
+
+#                 # assert expected calls
+#                 mock_robot_subject.assert_called_with(test_robot_serial_number, test_robot_uuid)
+#                 mock_dest_subject.assert_called_with(test_barcode)
+#                 mock_source_subject.assert_called_with("plate_abc", "bba490a1-9858-49e5-a096-ee386f99fc38")
+
+#                 root_sample_ids = ["sample_001", "sample_002"]
+#                 expected_samples = list(filter(lambda x: x[FIELD_ROOT_SAMPLE_ID] in root_sample_ids, samples))
+#                 # remove the microsecond comparing
+#                 for sample in expected_samples:
+#                     sample.update({FIELD_DATE_TESTED: sample[FIELD_DATE_TESTED].replace(microsecond=0)})
+#                 print(expected_samples)
+#                 assert len(root_sample_ids) == len(expected_samples)  # sanity check
+#                 for args, _ in mock_sample_subject.call_args_list:
+#                     # remove the microsecond before comparing
+#                     args[0][FIELD_DATE_TESTED] = args[0][FIELD_DATE_TESTED].replace(microsecond=0)
+#                     print(args[0])
+#                     print(expected_samples)
+#                     assert args[0] in expected_samples
+
+#                 # assert expected return values
+#                 assert len(errors) == 0
+#                 args, _ = mock_message.call_args
+#                 message_content = args[0]
+
+#                 assert message_content["lims"] == app.config["RMQ_LIMS_ID"]
+
+#                 event = message_content["event"]
+#                 assert event["uuid"] == str(test_uuid)
+#                 assert event["event_type"] == PLATE_EVENT_DESTINATION_FAILED
+#                 assert event["occured_at"] == test_timestamp
+#                 assert event["user_identifier"] == test_user
+
+#                 subjects = event["subjects"]
+#                 assert len(subjects) == 6
+#                 assert test_robot_subject in subjects  # robot subject
+#                 assert test_dest_subject in subjects  # destination plate subject
+#                 assert {  # control sample subject
+#                     "role_type": "control",
+#                     "subject_type": "sample",
+#                     "friendly_name": "positive control: 789_B01",
+#                     "uuid": str(test_uuid),
+#                 } in subjects
+#                 assert subjects.count(test_sample_subject) == 2  # sample subjects
+#                 assert test_source_subject in subjects  # source plate subject
+
+#                 metadata = event["metadata"]
+#                 assert metadata == {"failure_type": test_failure_type}
```

Comment:
We can remove this commented code if not needed

Reply:
Thanks, I need to double check why I commented this out.